Cargo.lock

```diff
@@ -2503,6 +2512,7 @@ dependencies = [
  "algebra",
  "anyhow",
  "chrono",
+ "chrono-humanize",
```

Comment:
Do we really need an extra crate to print the number of minutes/seconds? `chrono` itself is already a good additional to std types.

Reply:
it's easier for operators to read, but I'm willing to remove it if you think the cost is too high

Reply:
My concern here is not about compile times, but about readability. It's not a blocker for merging this PR though.

When it comes to reasoning about the code, the behaviour of
```rust
println!("{}m {}s", duration.num_minutes(), duration.num_seconds());
```
is more obvious than
```rust
println!("{}", HumanTime::from(duration));
```